src/main/java/com/utils/BasicTest.java

```diff
@@ -15,20 +15,14 @@ public abstract class BasicTest {
     
     public static final Logger logger = LogManager.getLogger();
     protected static WebDriver driver;
-    // private String driverPath;
 
     @BeforeMethod
     public void preCondition() {
-        // Chromedriver path
-        // driverPath = "src/main/resources/WebDrivers/chromedriver.exe";
-        // ChromeOptions options = new ChromeOptions();
-        // System.setProperty("webdriver.chrome.driver", driverPath);
-        // driver = new ChromeDriver(options);
+
         WebDriverManager.chromedriver().setup();
         driver = new ChromeDriver();
-        // Maximize the browser
         driver.manage().window().maximize();
-        //driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
```

Comment:
Nếu mình xài explicitly wait thì không  xai cái này. Remove it

Reply:
Đã remove